test/extended/operators/certs.go

```diff
@@ -49,6 +56,82 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
 
+	g.It("all tls artifacts must be known", func() {
+
+		ctx := context.Background()
+		kubeClient := oc.AdminKubeClient()
+		if ok, _ := exutil.IsMicroShiftCluster(kubeClient); ok {
+			g.Skip("microshift does not auto-collect TLS.")
+		}
+
+		nodes := map[string]int{}
+		controlPlaneLabel := labels.SelectorFromSet(map[string]string{"node-role.kubernetes.io/control-plane": ""})
+		nodeList, err := kubeClient.CoreV1().Nodes().List(ctx, metav1.ListOptions{LabelSelector: controlPlaneLabel.String()})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		for i, node := range nodeList.Items {
+			nodes[node.Name] = i
+		}
+
+		actualPKIContent, err := certgraphanalysis.GatherCertsFromPlatformNamespaces(ctx, kubeClient)
```

Comment:
if you layer `certsToRegistryInfo` with a function that takes `(pkiLists ...*PKIList)` and use that for the both this and  `GetPKIInfoFromRawData`, you'll be able to look at the resulting ownership lists instead of raw values.